src/BulkTools/HTTPBulkToolsResponse.php

```diff
@@ -362,18 +360,12 @@ protected function outputBody()
         $this->createBody();
         parent::outputBody();
     }
-
+    
     /**
-     * Catches fatal PHP error and output something useful for the front end
+     * This empty function exists only for semver
```

Comment:
There's a [proper way of marking a method as deprecated](https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc-tags.md#54-deprecated).

```suggestion
     * @deprecated 3.1.0 This function was used to catch PHP Errors and inject additional information in the response
```